docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -21,10 +21,41 @@ NOTE: KQL autocomplete for `.alerts-security.alerts-*` indices is available on t
 *Alerts* and *Rule details* pages, and in Timeline when either `All` or
 `Detection alerts` is selected.
 
+[role="screenshot"]
+image::images/rule-query.png[Shows how to search for a specific rule]
+
 * Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
+
++
+TIP: Use the Quick select menu to choose commonly or frequently used options.
++
+
+[role="screenshot"]
+image::images/update-timespan.gif[Shows how to update the time filter]
+
 * View detection alerts generated by a specific rule. Go to *Detect* -> *Rules*, then select a rule name in the All rules table. The rule details page displays a comprehensive view of the rule's settings, and the Alerts table under the Trend histogram displays the alerts associated with the rule, including alerts from any previous or deleted revision of that rule.
-* Group alerts by specific parameters and view individual alert counts by using the *Stack by* drop-down in the *Count* table. For example, if you select `kibana.alert.rule.name`, the table displays the number of alerts generated by each rule.
-* Visualize alerts by specific parameters by using the *Stack by* drop-down in the *Trend* histogram. For example, if you select `kibana.alert.rule.name`, the histogram displays the total alert counts by rule name.
+
+[role="screenshot"]
+image::images/rule-details.gif[Shows how to access alerts associated with a rule]
+
+* Group alerts by specific parameters and display individual alert counts by using the *Stack by* drop-down in the *Count* table. You can select or enter any aggregatable ECS field that is available on your alerts. For example, if you select `kibana.alert.rule.name`, the table displays the number of alerts generated by each rule.
```

Comment:
```suggestion
* Group alerts by specific parameters and display individual alert counts by using the *Stack by* drop-down in the *Count* table. You can select or enter any aggregatable ECS field that is available on your alerts. For example, if you select `kibana.alert.rule.name`, the table groups alerts by that field.
```

Reply:
While it does group the alert count by the field, I think I prefer the original because it describes what the user can expect to see, as opposed to how the table is aggregating alerts.